Makefile

```diff
@@ -1,5 +1,9 @@
 # Copyright 2016-2017 Palmer Dabbelt <palmer@dabbelt.com>
 
+# Disable the built-in rules to speed up execution.
+MAKEFLAGS += -r
```

Comment:
Yes, often when I'm debugging the makefile I'll put -d to see what rules it's running and having the -r reduces spam and makes the log a lot more readable.